<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BptSegmentForm</class>
 <widget class="QWidget" name="BptSegmentForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>610</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BPT Segment Form</string>
  </property>
  <widget class="QLabel" name="lblImage">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>60</y>
     <width>481</width>
     <height>381</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnFileBrowser">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>10</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>File browser</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblFileBrowser">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>55</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>File</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblClustering">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>590</y>
     <width>55</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Clustering</string>
   </property>
  </widget>
  <widget class="QLabel" name="lblNumberOfClass">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>590</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Class number</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="spinBox">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>580</y>
     <width>49</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>580</y>
     <width>61</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Cluster</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnTreeInspector">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>10</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Tree Inspector</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>221</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Segmentation by region number</string>
   </property>
   <widget class="QLabel" name="lblNumberOfRegions">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Region Number</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinNumberOfPruningRegion">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>49</width>
      <height>25</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
    <property name="value">
     <number>10</number>
    </property>
   </widget>
   <widget class="QPushButton" name="btnView">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>51</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>View</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>231</width>
     <height>261</height>
    </rect>
   </property>
   <property name="title">
    <string>Segmentation by Evolvement</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <widget class="QPushButton" name="btnNoThreshold">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>40</y>
      <width>85</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>No Threshold</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtTextureWavelet">
    <property name="geometry">
     <rect>
      <x>162</x>
      <y>110</y>
      <width>41</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>0.5</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtMeanGradient">
    <property name="geometry">
     <rect>
      <x>162</x>
      <y>140</y>
      <width>41</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>0.5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAutoPruning">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>40</y>
      <width>85</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Auto Pruning</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtEvolvementThreshold">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>41</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>0.5</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkTextureWavelet">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>110</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Texture Wavelet</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtMeanColor">
    <property name="geometry">
     <rect>
      <x>162</x>
      <y>80</y>
      <width>41</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>0.5</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkHomogeneity">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>200</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Homogeneity</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtHomogeneity">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>200</y>
      <width>41</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>0.5</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblEvolvementThreshold">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Evolvement Threshold</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkMeanColor">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>80</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Mean color</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkMeanGradient">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>140</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Mean Gradient</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtRegionSize">
    <property name="geometry">
     <rect>
      <x>162</x>
      <y>170</y>
      <width>41</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>0.5</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkRegionSize">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>170</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Region Size</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkColorVariance">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>230</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Color Variance</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtColorVariance">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>230</y>
      <width>41</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>0.5</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>370</y>
     <width>231</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>Classification after segmentation</string>
   </property>
   <widget class="QPushButton" name="btnShowManualClassification">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>111</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Manually Classify</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblLearningFile">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Learning File</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnLearningFile">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>71</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnClassify">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>70</y>
      <width>51</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Classify</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnClassificationMethod">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>70</y>
      <width>85</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Method</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>470</y>
     <width>211</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>Validation with Seg file</string>
   </property>
   <widget class="QPushButton" name="btnSegFileBrowser">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>101</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Seg File Browser</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCompaire">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>85</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Compaire</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>530</y>
     <width>211</width>
     <height>61</height>
    </rect>
   </property>
   <property name="title">
    <string>Segmentation after classification</string>
   </property>
   <widget class="QPushButton" name="btnClassSegment">
    <property name="geometry">
     <rect>
      <x>84</x>
      <y>20</y>
      <width>111</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Class Segment</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btnTreeViewer">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>10</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Tree Viewer</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSaveImage">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>10</y>
     <width>85</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Save Image</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
